lib/feature_flagger/configuration.rb

```diff
@@ -0,0 +1,37 @@
+module FeatureFlagger
+  class Configuration
+    attr_reader :storage, :yaml_filepath, :info
+
+    def initialize
+      configure_storage
+      configure_yaml_filepath
+      configure_info
+    end
+
+    def storage=(storage)
+      @storage = storage.nil? ? default_client : storage
```

Comment:
Is there the need of assign the default_storage here since it is assigned in `#configure_storage`?